pkg/api/types.go

```diff
@@ -1664,6 +1667,104 @@ type SecretList struct {
 	Items []Secret `json:"items"`
 }
 
+// SecurityContext holds security configuration that will be applied to a container.  If a security context
+// is set on the container spec then it must comply with any constraints defined in the SecurityConstraints context
```

Comment:
Maybe SecurityConstraints and its dependencies are part of the kubelet API but not necessarily part of the core kubenetes API?  Take a look at how scheduler has its own API: 
plugin/pkg/scheduler/api/...

SecurityContext seems like it is part of the core API since Pods use it.